cgo/interface.go

```diff
@@ -0,0 +1,37 @@
+package cgo
```

Comment:
looks like a clean copy of https://github.com/filecoin-project/filecoin-ffi/blob/145f6258251d257749d5aa953c98dd0364231f48/cgo/interface.go 👍